src/manifest.json

```diff
@@ -47,7 +47,12 @@
     "browser_style": true,
     "default_icon": "img/container-site.svg",
     "default_title": "Multi-Account Containers",
-    "default_popup": "popup.html"
+    "default_popup": "popup.html",
+    "theme_icons": [{
+        "light": "img/container-site-light.svg",
+        "dark": "img/container-site.svg",
+        "size": 32
```

Comment:
Note: Size is mandatory according to [this](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_action#theme_icons). I don't know why thought.